decs.d.ts

```diff
@@ -15,4 +20,104 @@ declare module '@gridsuite/commons-ui' {
     }
 
     export function useSnackMessage(): UseSnackMessageReturn;
+
+    type Input = string | number;
+    type Options = Array<{
+        id: string;
+        label: string;
+    }>;
+
+    interface AutocompleteInputProps
+        extends Omit<
+            AutocompleteProps<
+                string,
+                boolean | undefined,
+                boolean | undefined,
+                boolean | undefined
+            >,
+            'renderInput' // we already defined it in our custom Autocomplete
+        > {
+        name: string;
+        label?: string;
+        options: Array<any>; // https://mui.com/material-ui/react-autocomplete/#options-structure
+        outputTransform?: (value: string) => string;
+        inputTransform?: (value: string) => string;
+        readOnly?: boolean;
+        previousValue?: string;
+        allowNewValue?: boolean;
+        onChangeCallback?: () => void;
+        formProps?: TextFieldProps;
+    }
+
+    export function AutocompleteInput(props: AutocompleteInputProps);
+
+    interface ErrorInputProps {
+        name: string;
+        InputField?: ReactElement;
+    }
```

Comment:
```suggestion
    interface ErrorInputProps {
        name: string;
        InputField?: ReactNode;
    }
```

now FC returns a ReactNode instead of ReactElement, I think we should use ReactNode as well

Reply:
Actually after few research I think it's not, in our case it can only be a ReactElement

Reply:
[ReactNode is wider, it can be text, number, boolean, null, undefined, a portal, a ReactElement, or an array of ReactNodes. It represents anything that React can render.](https://stackoverflow.com/a/72353143)